atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java

```diff
@@ -1230,7 +1233,7 @@ public void compactInternally(TableReference tableRef) {
     }
 
     private void alterTableForCompaction(TableReference tableRef, int gcGraceSeconds, float tombstoneThreshold) {
-        log.trace("Altering table {} to have gc_grace_seconds={} and tombstone_threshold=%.2f",
+        log.trace("Altering table {} to have gc_grace_seconds={} and tombstone_threshold={}",
```

Comment:
We're losing the formatting here, so may want to manually format the `tombstoneThreshold` arg.